__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const app = require('../app.js')
 const seed = require('../db/seeds/seed.js')
 const db = require('../db/connection.js')
 const testData = require('../db/data/test-data/index.js')
+const { json } = require('express')
```

Comment:
This should be in the `./app.js` module. See these [notes](https://notes.northcoders.com/courses/js-back-end/middleware#expressjson)